lnd-rpc/lnd-rpc.sbt

```diff
@@ -0,0 +1,59 @@
+import java.nio.file._
+import scala.util.Properties
+
+name := "bitcoin-s-lnd-rpc"
+
+libraryDependencies ++= Deps.lndRpc
+
+CommonSettings.prodSettings
+
+enablePlugins(AkkaGrpcPlugin)
+
+// Disable fatal warning otherwise protobuf deprecation warnings will cause errors
+scalacOptions in Compile ~= (_ filterNot (s => s == "-Xfatal-warnings"))
```

Comment:
You should be able to filter this by using the same tools I used in this PR (specifically ignoring errors in the `protobuf` package)

https://github.com/bitcoin-s/bitcoin-s/pull/2822

Reply:
Doesn't seem you can disable deprecation warnings for a single package

Reply:
Here is the incantation you need

```scala
scalacOptions in Compile += {
  //Added to supress all of the deprecation warnings inside of the
  //generated scala code by the grpc plugin. Two packages have deprecations
  //inside of them, 'signrpc' and 'lnrpc'
  //see: https://www.scala-lang.org/2021/01/12/configuring-and-suppressing-warnings.html
  if (scalaVersion.value.startsWith("2.13")) {
    "-Wconf:cat=deprecation:site=lnrpc\\..*:silent,cat=deprecation:site=signrpc\\..*:silent"
  } else {
    //-Wconf doesn't work on Scala 2.12.x until we get 2.12.13
    //which is currently blocked on a scoverage bug
    ""
  }
}
```